pages/tasks/team_details.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+from pages.tasks.regions.task import Task
+
+
+class TeamDetailsPage(Base):
+
+    _name_locator = (By.CSS_SELECTOR, '.content-container > h2')
+    _task_list_header_locator = (By.CSS_SELECTOR, 'div.task-list-container > h3')
```

Comment:
You could probably leave the `div` off from this locator which would make it less fragile.